src/utils/index.js

```diff
@@ -193,6 +193,25 @@ export default {
     return true
   },
 
+  regexCompare (value, search) {
+    try {
+      const regexpParts = search.match(/^\/(.*?)\/([gim]*)$/)
+      let regex
+
+      if (regexpParts) {
+        regex = new RegExp(regexpParts[1], regexpParts[2])
+      } else {
+        regex = new RegExp(search, 'gim')
+      }
+
+      if (value.toString().search(regex) !== -1) {
```

Comment:
Can be 
```js
if (value.toString().search(regex) > -1) {
```

Reply:
`!==` is more strict, so it's ok i guess.
But if you wish i also can change it to `>`.

Reply:
Let's leave it as !==.. It's just the way we code

Reply:
We often use that in the bootstrap-table as example here: https://github.com/wenzhixin/bootstrap-table/blob/develop/src/bootstrap-table.js#L448
So lets leave it as `!==`.